doc/api/child_process.md

```diff
@@ -1094,6 +1094,27 @@ console.log(`Spawned child pid: ${grep.pid}`);
 grep.stdin.end();
 ```
 
+### subprocess.ref()
+<!-- YAML
+added: v0.7.0
```

Comment:
Shouldn't this be `0.7.10`? And same for `unref`.

Reply:
Whoops! Will fix.